xml_inscriptions/transcribed/IN.Bloom.IUAM.G.63.105.84f.xml

```diff
@@ -34,7 +34,7 @@
                   <objectDesc ana="#object_other">
                      <supportDesc ana="#material_other">
                         <support>
-                           <p>translucent carnelian </p>
+                           <p>Carnelian gemstone</p>
```

Comment:
I'd keep the detail about it being translucent

Reply:
Related to the previous discussion about extraneous detail, this can stay as you've adjusted it.